pages/common/tree.md

```diff
@@ -14,14 +14,14 @@
 
 `tree -a`
 
-- Print human readable size of files:
+- Print the tree without indentation lines, showing the full path instead:
 
-`tree -h`
+`tree -i -f`
 
-- Print the full path for each file:
+- Print the size of each node next to it, in human-readable format, with folders displaying their cumulative size (as in the `du` command):
 
-`tree -f`
+`tree -s -h --du`
 
-- Print the tree without lines and indentation. Useful when used with -f:
+- Filter the tree using a wildcard (glob) pattern, showing only the hierarchies leading to the matching entries:
 
-`tree -i`
+`tree -P {{*.txt}} --prune`
```

Comment:
@agnivade @ostera actually, the `--prune` option is not implemented in linux's `tree`. Per the [manpage](http://linux.die.net/man/1/tree):

> ### Bugs

> Tree does not prune "empty" directories when the -P and -I options are used. Tree prints directories as it comes to them, so cannot accumulate information on files and directories beneath the directory it is printing.

That page also shows that the `--du` option does not exist for the linux version.

So I guess I'll change this entry to be simply `tree -P {{*.txt}}` and the one before it to be `tree -s -h`, then move the page to linux/tree.md, and create a separate osx/tree.md for the behavior including `--prune` and `--du`. Sounds good?